www/index.html

```diff
@@ -468,10 +468,13 @@ <h3>Display settings</h3>
                                     <select class="form-control" id="appThemeSelect">
                                         <option value="light">Light</option>
                                         <option value="dark">Dark (app only)</option>
-                                        <option value="dark_invert">Dark + invert (generic)</option>
-                                        <option value="dark_mwInvert">Dark + invert (Wikimedia)*</option>
+                                        <option value="dark_invert">Dark (generic invert)</option>
+                                        <option value="dark_mwInvert">Dark (wikimedia invert)*</option>
+                                        <option class="auto" value="auto">Auto (app only)</option>
+                                        <option class="auto" value="auto_invert">Auto (generic invert)</option>
+                                        <option class="auto" value="auto_mwInvert">Auto (wikimedia invert)*</option>
                                     </select>
-                                    <p id="dark_mwInvert-help" class="appThemeInfo form-text text-muted">* Implements workarounds specific to Wikimedia ZIMs. Try generic option if there are display errors with recent ZIMs.</p>
+                                    <p id="_mwInvert-help" class="appThemeInfo form-text text-muted">* Implements workarounds specific to Wikimedia ZIMs. Try generic option if there are display errors with recent ZIMs.</p>
```

Comment:
changed from `dark_mwInvert` to` _mwInvert` , to get the wikimedia help for whenevr the wikimedia content theme is applied.(purpose : scope)